src/traces/parcoords/parcoords.js

```diff
@@ -647,7 +649,12 @@ module.exports = function parcoords(gd, cdModule, layout, callbacks) {
         .attr('stroke-width', '1px');
 
     axis.selectAll('text')
-        .style('text-shadow', '1px 1px 1px #fff, -1px -1px 1px #fff, 1px -1px 1px #fff, -1px 1px 1px #fff')
+        .style('text-shadow',
+            paperColor + ' 1px 1px 1px, ' +
+            paperColor + ' -1px -1px 1px, ' +
+            paperColor + ' 1px -1px 1px, ' +
+            paperColor + ' -1px 1px 1px'
+        )
```

Comment:
Great. This logic is similar to what we have for `parcats`.
Also It seems we need to change `sankey` here:
https://github.com/plotly/plotly.js/blob/f125c2482737ea37ac0b0ccdd294ce99804e3dc2/src/traces/sankey/render.js#L1026-L1028

If so, could you make a little helper function to handle `1px` `text-shadows` please?

Reply:
Done in https://github.com/plotly/plotly.js/pull/5506/commits/bf065001a65e6158347d541f8608da2a1b56b84d